front/lib/api/assistant/global_agents.ts

```diff
@@ -133,7 +133,7 @@ function _getDefaultSearchActionForGlobalAgent({
     sId: agentSid + "-search-action",
     type: "websearch_configuration",
     name: DEFAULT_WEBSEARCH_ACTION_NAME,
-    description: `Search the web for more information.`,
+    description: `Perform a web search.`,
```

Comment:
Can we create a global variable for this one that we have at least in 2 places already?

We can store it in DEFAULT_WEBSEARCH_ACTION_DESCRIPTION in the same file as DEFAULT_WEBSEARCH_ACTION_NAME